account_manager/src/validator/import.rs

```diff
@@ -169,7 +169,15 @@ pub fn cli_run(matches: &ArgMatches, validator_dir: PathBuf) -> Result<(), Strin
         let password_opt = loop {
             if let Some(password) = previous_password.clone() {
                 eprintln!("Reuse previous password.");
-                break Some(password);
+                if check_password_on_keystore(&keystore, &password)? {
+                    break Some(password);
+                } else {
+                    eprintln!("Reused password incorrect. Retry!");
+                    if let Some(password) = get_password(stdin_inputs) {
+                        previous_password = Some(password.clone());
+                        break Some(password);
```

Comment:
The current issue is that upon a reused password, after prompting the user to enter the password (line 177), it does not check whether the password is correct or not and proceed to the next keystore regardless. So I added a call to `check_password_on_keystore` to check the password. 
```suggestion
                        previous_password = Some(password.clone());
                        if check_password_on_keystore(&keystore, &password)? {
                        break Some(password);
                        }
```
The issue now is, because it becomes a nested if loop, the UX when the password is incorrect becomes:
`Invalid password`
at a first try, and then jumps to the beginning of the loop and display:
```
Invalid password
Reuse previous password.
Invalid password
Reused password incorrect. Retry!
```
and alternating that way until a correct password is entered. Do we have a better way for the UX? I try using a while loop but that's weird outcome too

Reply:
Fixed in [bd035f9](https://github.com/sigp/lighthouse/pull/4380/commits/bd035f9f5b3cc4f14e549ca2f2795c7b76ebea3a)

Thanks for catching it and testing the fix CK :)